packages/nuxt/src/module.ts

```diff
@@ -84,6 +84,26 @@ export default defineNuxtModule<ModuleOptions>({
           });
         }
 
+        consoleSandbox(() => {
+          const serverDir = nitro.options.output.serverDir;
+
+          // Netlify env: https://docs.netlify.com/configure-builds/environment-variables/#build-metadata
+          if (serverDir.includes('.netlify') || !!process.env.NETLIFY) {
+            // eslint-disable-next-line no-console
+            console.warn(
+              '[Sentry] Warning: The Sentry SDK detected a Netlify build. Server-side support for the Sentry Nuxt SDK on Netlify is currently unreliable due to technical limitations of serverless functions. Traces are not collected, and some errors may not be reported. For more information on setting up Sentry on the Nuxt server-side, please refer to the documentation: https://docs.sentry.io/platforms/javascript/guides/nuxt/install/',
+            );
+          }
+
+          // Vercel env: https://vercel.com/docs/projects/environment-variables/system-environment-variables#VERCEL
+          if (serverDir.includes('.vercel') || !!process.env.VERCEL) {
+            // eslint-disable-next-line no-console
+            console.warn(
+              '[Sentry] Warning: The Sentry SDK detected a Vercel build. The Sentry Nuxt SDK support for ESM on Vercel is currently experimental. Traces are not collected, but errors are reported. For more information on setting up Sentry on the Nuxt server-side, please refer to the documentation: https://docs.sentry.io/platforms/javascript/guides/nuxt/install/',
```

Comment:
I am wondering, isn't Nuxt always ESM? If that is the case, it may make sense then to just not call out ESM at all and just say that tracing doesn't work on Vercel:

```suggestion
              '[Sentry] Warning: The Sentry SDK detected a Vercel build. The Sentry Nuxt SDK currently does not support tracing on Vercel. For more information on setting up Sentry on the Nuxt server-side, please refer to the documentation: https://docs.sentry.io/platforms/javascript/guides/nuxt/install/',
```

Reply:
Yes it's always ESM - do you think it would still make sense to include this information somewhere? Like "The Sentry SDK detected a Vercel build. The Sentry Nuxt SDK currently does not support tracing on Vercel because of ESM on the server-side."?

Reply:
From a user's perspective, I wouldn't care, since there is no way I can fix it. No strong opinions though. If you feel like it adds value, feel free to add a sentence like that.